### ISSUE TITLE
Wording: deactivating a user

### ISSUE THREAD

**Issue by GabyKuster** [dasch-swiss/knora-app#202]
1. I am logged in as a sys admin
2. I wish to deactivate a user
3. In the ... - menu I get the option: "suspend user"
4. When I click the option, the opening window has the name "delete user" and asks: "do you want to delete this user" I ahve no idea now, if the user is deactivated or really no longer in the system.

Please keep all the "deactication"-Actions consistent throughout the application and distinguish clearly when something is "deleted" meaning gone forever, "deactivated " =cannot be accessed or seen by ordinary users and "archived" = no change of system data possible".

Of course the  naming can be different, but it must me consistent throughout Knora-app and need to be clearly explained either in text or tool-tips.

**OPTIONAL: Expected behavior**

**Screenshots**
If applicable, add screenshots to help explain your problem (drag-and-drop the image directly here).

**Desktop (please complete the following information):**
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

**Additional context**

**Comment by kilchenmann** [comment]
resolved in #218

**Comment by subotic** [comment]
Deactivate:
  - put user in deactivated state
  - not allow user to do anything (login, create something, etc.)

Delete:
  - delete user from database
  - will only be possible if user is not referenced somewhere in the data
  - currently not possible as it is not implemented in `knora-api` but will be in the future

Suspend should be changed (for now) to deactivate. A tooltip can provide the necessary explanation.

**Comment by kilchenmann** [comment]
Thanks @subotic, but we changed the wording already twice. From deactivate to suspend, back to deactivate and after my comments back again to suspend. The last change is already merged in PR #218. Suspend is similar to deactivate and means the same functionality. I've adopted it from other software platforms. In the GUI we shouldn't always use the same terminology as in knora-api. For example we don't use the word ontology, but data model. There are always small differences, but with better understanding for the users. Yes, deactivate is a clear word. But we shouldn't spend to much time in renaming all the time. We have to work on the functionality now.
